### ISSUE TITLE
IO error with import threeML

### ISSUE THREAD

**Issue by rupalbasak** [giacomov/3ML#239]
I installed 3ML in my laptop running Ubuntu 14.04. 
I used sudo pip install for this purpose. No error during installation.
Then I used "jupyter notebook" to open the localhost in chrome.
But, when I started a new project with python 2, the "import threeML" gives the 
following i/o error. Please suggest.

IOError                                   Traceback (most recent call last)
<ipython-input-1-363e3fd550d8> in <module>()
----> 1 import threeML

/usr/local/lib/python2.7/dist-packages/threeML/__init__.py in <module>()
      4 
      5 # Finally import the serialization machinery
----> 6 from .io.serialization import *
      7 
      8 from .exceptions.custom_exceptions import custom_warnings

/usr/local/lib/python2.7/dist-packages/threeML/io/serialization.py in <module>()
----> 1 from threeML.classicMLE.joint_likelihood import JointLikelihood
      2 from threeML.bayesian.bayesian_analysis import BayesianAnalysis
      3 
      4 __all__ = []
      5 

/usr/local/lib/python2.7/dist-packages/threeML/classicMLE/joint_likelihood.py in <module>()
      7 
      8 import numpy as np
----> 9 import astromodels.core.model
     10 
     11 from threeML.minimizer import minimization

/usr/local/lib/python2.7/dist-packages/astromodels/__init__.py in <module>()
      8 if os.environ.get("ASTROMODELS_DEBUG", None) is None:
      9 
---> 10     from .sources.point_source import PointSource
     11     from .sources.extended_source import ExtendedSource
     12     from .sources.particle_source import ParticleSource

/usr/local/lib/python2.7/dist-packages/astromodels/sources/point_source.py in <module>()
      1 import collections
      2 
----> 3 import astropy.units as u
      4 import numpy
      5 

/usr/local/lib/python2.7/dist-packages/astropy/__init__.py in <module>()
    287     log = _init_log()
    288 
--> 289     _initialize_astropy()
    290 
    291     from .utils.misc import find_api_page

/usr/local/lib/python2.7/dist-packages/astropy/__init__.py in _initialize_astropy()
    209 
    210     try:
--> 211         config.configuration.update_default_config(__package__, config_dir)
    212     except config.configuration.ConfigurationDefaultMissingError as e:
    213         wmsg = (e.args[0] + " Cannot install default profile. If you are "

/usr/local/lib/python2.7/dist-packages/astropy/config/configuration.pyc in update_default_config(pkg, default_cfg_dir_or_fn, version)
    703     if doupdate or needs_template:
    704         if needs_template:
--> 705             with io.open(template_path, 'wt', encoding='latin-1') as fw:
    706                 fw.write(template_content)
    707             # If we just installed a new template file and we can't

IOError: [Errno 13] Permission denied: '/home/rupal/.astropy/config/astropy.2.0.1.cfg'

**Comment by giacomov** [comment]
Hi,
it looks like your astropy configuration has the wrong permits. Probably it was created by root.

To avoid problems like these, I suggest you to install 3ML following the instructions (or even better the installation script). You do not need to be root to install 3ML, and actually it is better if you are not as you have seen.

To fix your problem, just become root once again issuing this command:

```
sudo chown rupal:rupal /home/rupal/.astropy/config/astropy.2.0.1.cfg
```

and try again.

However, I suggest you to uninstall threeML from the system root like this:

```
sudo pip uninstall threeML astromodels
sudo pip uninstall threeML astromodels
```

 and re-install it following the instructions you can find in the home page here on github.

Let me know if that works,

**Comment by rupalbasak** [comment]
Hi,

Thanks for your reply. I indeed tried in the beginning to install the software using the python script. It did install, but the jupyter notebook was not working. My colleague told me that he had the same problem, so I installed it using the other method. But, without root, I could not install. Then I tried to install with sudo pip. 

Now, about your suggestion, i uninstalled and installed again with sudo pip. Then I did  
sudo chown rupal:rupal /home/rupal/.astropy/config/astropy.2.0.1.cfg

But, got the error that there is no such file.
chown: cannot access ‘/home/rupal/.astropy/config/astropy.2.0.1.cfg’: No such file or directory

Best regards,
Rupal
